zerver/models.py

```diff
@@ -268,7 +268,7 @@ class Realm(models.Model):
     WILDCARD_MENTION_POLICY_ADMINS = 5
     WILDCARD_MENTION_POLICY_NOBODY = 6
     wildcard_mention_policy: int = models.PositiveSmallIntegerField(
-        default=WILDCARD_MENTION_POLICY_STREAM_ADMINS,
+        default=WILDCARD_MENTION_POLICY_ADMINS,
```

Comment:
I'm not sure we need to do this -- it might be cleaner to do a hack in how the feature is described to users for the intermediate period.

Reply:
Also, I was thinking whether it will be better to keep ADMINS only as default instead of STREAM_ADMINS, as per our current approach, we would make the user creating the stream a stream admin, so such users would also be allowed to use wildcard mentions in those streams.

Reply:
Yeah, I was actually just thinking about that question.  I think that the right way to stop that sort of abuse is to restrict who can create streams (which we recommend in our "moderating open organizations" document) for that sort of public community.

(Creating a stream and then subscribing other users to it is disruptive in the same way that a wildcard mention is).